service/project.go

```diff
@@ -366,6 +367,19 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 			uis.LoggedError(w, r, http.StatusBadRequest, errors.Errorf("Cannot enable Commit Queue without github webhooks enabled for the project"))
 			return
 		}
+		if len(responseRef.CommitQueueAliases) == 0 {
+			aliases, err := model.FindAliasInProject(responseRef.Identifier, evergreen.CommitQueueAlias)
```

Comment:
if there are preexisting commit queue aliases and the user has deleted them in this modifyProject request I think we will not error since they will still be found (they're only deleted later)
https://github.com/evergreen-ci/evergreen/blob/d5aed3f8061abe38fc93f53b836f1693eccbb651/service/project.go#L523